jujugui/static/gui/src/app/app.js

```diff
@@ -728,8 +728,8 @@ YUI.add('juju-gui', function(Y) {
       @method _renderUserProfile
     */
     _renderUserProfile: function() {
-      var user = this.get('users')['charmstore'];
-      var username = user && user.user && user.user.name;
+      var auth = this.get('users')['charmstore'];
+      var username = auth && auth.user && auth.user.name;
```

Comment:
I hit a conflict here because we had pretty much *exactly* the same code. The only diff was that I went with `auth` specifically to avoid `user.user` ;-)